### PR TITLE
core(bootup-time): refactor task/group iteration

### PR DESCRIPTION
**Pull request by paulirish**

### CODE REVIEW

File: lighthouse-core/audits/bootup-time.js [hunk 1]
```diff
@@ -114,22 +114,28 @@ class BootupTime extends Audit {
...
+        // eventStyle() returns a string like 'Evaluate Script'
+        const task = WebInspector.TimelineUIUtils.eventStyle(perTaskPerUrlNode.event);
+        // Resolve which taskGroup we're using
+        const groupName = taskToGroup[task.title] || group.other;
+        const groupTotal = taskGroups[groupName] || 0;
+        taskGroups[groupName] = groupTotal + (perTaskPerUrlNode.selfTime || 0);
       });
-      result.set(url, tasks);
+      Object.keys(taskGroups).forEach(groupName => {
+        taskGroups[groupName] = Math.round(taskGroups[groupName] * 10) / 10;
```
Comment (wardpeet) [inline lighthouse-core/audits/bootup-time.js]: Why do you want to reduce the number after the period?

Comment (paulirish) [reply lighthouse-core/audits/bootup-time.js]: good call. nuked this.

**Comment by paulirish** [comment]
> Do we want to sort this table as well? 

done.

**Comment by paulirish** [comment]
landed in 33b1574da7a404e2ad701ea9651b08a6e40245c7

**Pull request closed without merging**